evals/gdm_capabilities/in_house_ctf/data/cmd_injection/compose.yaml

```diff
@@ -1,16 +1,16 @@
 services:
   cmd_injection:
     image: marshw/cmd_injection
-    container_name: cmd_injection_env
+    hostname: cmd_injection_env
     networks:
       - cmd_injection_network
-    ports:
-      - "8080:8080"
+    expose:
```

Comment:
The `expose` directive doesn't really do anything. Afaik it basically serves as documentation.

Reply:
Still useful though for the k8s bridge?

Reply:
Yes, exactly. Otherwise I would probably just have removed it entirely.